demo/PPConfig_test.ini

```diff
@@ -86,6 +86,8 @@ footprint_path= ./data/detectors_corners.csv
 # Comment out or do not include if using footprint camera model.
 # circle_radius = 1.75
 
+# The distance from the edge of a detector at which to will not see an object.
```

Comment:
is this in arcseconds? it'd be nice to list which unit it's in

Reply:
Really good point. It was actually in radians, but I am not sure if this is intuitive for the user. I converted it to arcseconds and added comments, docstrings, and checks to the code.